packages/esm-patient-conditions-app/src/conditions/conditions-form.workspace.tsx

```diff
@@ -52,7 +55,7 @@ const ConditionsForm: React.FC<ConditionFormProps> = ({
             : null
           : null,
       conditionName: '',
-      clinicalStatus: condition?.cells?.find((cell) => cell?.info?.header === 'clinicalStatus')?.value ?? '',
+      clinicalStatus: formContext === 'editing' ? matchingCondition?.clinicalStatus?.toLowerCase() : '',
```

Comment:
The `clinicalStatus` value wasn't correctly getting picked when editing a condition. This fixes that.